scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -24,14 +24,27 @@
 if not ZEPHYR_BASE:
     sys.exit("$ZEPHYR_BASE environment variable undefined")
 
+try:
+    subproc = subprocess.run(['west', 'topdir'], check = True, stdout=subprocess.PIPE)
```

Comment:
If I remember correctly, check=True means exception is raised when exitcode !=0. That means `if subproc.returncode == 0:` is always true